go.sum

```diff
@@ -0,0 +1,370 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
no worries if you don't know, but how did this library even work w/o any go.mod or deps?

Reply:
`go mod init` and never think about the package management of yore if you want a happy life.